src/deepEqual.js

```diff
@@ -21,6 +21,20 @@
  */
 function deepEqual(a, b) {
   // write code here
+  let isNotObject = arg => typeof arg !== 'object' || arg === null;
+  let propsLength = (arr1, arr2) => Object.keys(arr1).length !== Object.keys(arr2).length;
```

Comment:
this function returns boolean. functions returning boolean are typically named 'isBlahBlah'. 

in this case arePropsLengthEqual would be a good name.